### PR TITLE
[TEVA-1387] - Exclude folders from triggering certain GitHub workflows

### PR DESCRIPTION
**Pull request by nickromneydfe**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-1387

## Changes in this PR:

- Exclude 5 folders not required to build the app from starting `deploy*` or `test` workflows

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ on:
   push:
     branches:
       - master
+    paths:
+    - '!.github/**'
```
Comment (saliceti) [inline .github/workflows/deploy.yml:8]: If we make a change to the workflow, we want to run it I guess, so this shouldn't be excluded?

Comment (nickromneydfe) [reply .github/workflows/deploy.yml:8]: My thinking was that as the workflow YML file itself is not part of the eventual app, then the next push which has a change to the app would be deployed using the new workflow. Possibly this decouples the feedback loop, so I'm happy to remove it.

Comment (saliceti) [reply .github/workflows/deploy.yml:8]: Yes I saw that issue in my previous job and it was really annoying.

Comment (nickromneydfe) [reply .github/workflows/deploy.yml]: Removed!

**Review by saliceti — APPROVED** [review]

**Pull request merged**